Dockerfile

```diff
@@ -34,6 +34,8 @@ RUN CYPRESS_INSTALL_BINARY=0 npm ci --no-audit && npm run build  && \
     export GIT_REF=${GIT_REF} && \
     npm run record-build-info
 
+RUN npm prune --no-audit --production
```

Comment:
what's the reason for moving this here?

Reply:
Sorry, I've added the narrative about this in 9d366b4ab1cb1e4928cf597da9785a70f111ffba:

> Prune production dependencies in the build stage
>
> Removing the packages in the final stage will not result in size savings
> as COPY layer will be retained: we need to remove the unneeded packages
> in the `build` phase so the `COPY --from=build` commands copy less.

Reply:
The difference in size between current `main` and this branch's Docker build:
```
$ docker images | grep 'ui-'
ui-before  latest                  6b974cc88063        2 seconds ago       425MB
ui-after   latest                  250473b4e6a7        2 minutes ago       226MB
```